asf_floorplan_interpreter/config/base.yaml

```diff
@@ -12,12 +13,14 @@ labels_to_predict:
     "OTHER",
   ]
 room_model_name: "room_config_yolov8m"
-window_door_model_name: "window_door_config_yolov8m_wd"
+window_door_model_name: "window_door_types_roboflow_config_yolov8m_wd"
 staircase_model_name: "staircase_config_yolov8m"
-room_type_model_name: "room_type_config_yolov8m"
-eval_data_file: "data/annotation/evaluation/Econest_test_set_floorplans_211123.csv"
+room_type_model_name: "room_type_bw_config_yolov8m"
+eval_data_file: "data/annotation/evaluation/Econest_test_set_floorplans_211223.csv"
```

Comment:
We should remove specific files names from the public repo I think (/anonymise at minimum)